### PR TITLE
feat: empty articles tab

### PR DESCRIPTION
**Pull request by alfonsobries**
## Summary

only adds an "empty" message, does not include search input and controls since that is subject for another task

https://app.clickup.com/t/8678v0h61

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] I added a storybook entry for the component that was added _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/js/icons.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ import { ReactComponent as Metamask } from "@icons/metamask.svg";
 import { ReactComponent as Minus } from "@icons/minus.svg";
 import { ReactComponent as DiamondOpacity } from "@icons/opacity/diamond.svg";
 import { ReactComponent as ImageOpacity } from "@icons/opacity/image.svg";
+import { ReactComponent as NewspaperOpacity } from "@icons/opacity/newspaper.svg";
```
Comment (ItsANameToo) [inline resources/js/icons.tsx:73]: why is this inside the `opacity` folder?

**Review by ItsANameToo — CHANGES_REQUESTED** [review]
this looks odd as the scrollable area has margins 🤔 would be better if it goes off screen 
image

File: resources/js/icons.tsx (hunk 1 above)
Comment (alfonsobries) [reply resources/js/icons.tsx:73]: @ItsANameToo really because rest of the icons on that tab are there, didnt really checked if it has opacity

**Comment by alfonsobries** [comment]
@ItsANameToo ✅

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**